env0/provider.go

```diff
@@ -127,7 +128,13 @@ func configureProvider(version string, p *schema.Provider) schema.ConfigureConte
 		if err != nil {
 			return nil, diag.Diagnostics{diag.Diagnostic{Severity: diag.Error, Summary: err.Error()}}
 		}
+		apiClient := client.NewApiClient(httpClient)
+		if apiClient == nil {
+			return nil, diag.Diagnostics{diag.Diagnostic{Severity: diag.Error, Summary: "Failed to fetch organiztions."}}
+		}
 
-		return client.NewApiClient(httpClient), nil
+		apiClient.Organization()
+		log.Printf("organizations fetched to cache Auth0 API response.")
```

Comment:
prepend [INFO].

`log.Println("[INFO] Resolving api key by name: ", id)`

I do wonder if this should be a log message or instead just a comment in the code....

Reply:
I logged only for my debugging, I can remove it

Reply:
[8257409](https://github.com/env0/terraform-provider-env0/pull/456/commits/8257409ecc7aa4e95734b4fcd4bda992911f18d8)